kotest-assertions/kotest-assertions-core/src/jvmMain/kotlin/io/kotest/matchers/resource/byteArrayMatchers.kt

```diff
@@ -53,8 +53,8 @@ fun ByteArray.shouldNotMatchResource(
 }
 
 fun matchResource(
-   resourcePath: String,
-   matcherProvider: (ByteArray) -> Matcher<ByteArray>,
+  resourcePath: String,
+  matcherProvider: (ByteArray) -> Matcher<ByteArray>,
```

Comment:
```suggestion
   resourcePath: String,
   matcherProvider: (ByteArray) -> Matcher<ByteArray>,
```

The indent should be 3 spaces, right?